src/main/java/cc/hyperium/config/Settings.java

```diff
@@ -502,6 +502,9 @@
     @ToggleSetting(name = "Item Physics", category = GENERAL)
     public static boolean ITEM_PHYSIC_ENABLED = false;
 
+    @ConfigOpt
+    public static long TOTAL_PLAYTIME = 0;
```

Comment:
Is a total like this necessary? Session time is a nick tracker, total seems a bit excessive

Reply:
unresolved [previously resolved by "sk1er-cant-code"]
let the people who are actually relevant in a situation resolve stuff ok people?